pywb/rewrite/html_rewriter.py

```diff
@@ -177,7 +177,7 @@ def _rewrite_srcset(self, value, mod=''):
             return ''
 
         values = (url.strip() for url in re.split(self.SRCSET_REGEX, value) if url)
-        values = [self._rewrite_url(v.strip()) for v in values]
+        values = [self._rewrite_url(v.split(' ')[0].strip()) + (' ' + ' '.join(v.split(' ')[1:])).rstrip() for v in values if v]
```

Comment:
It would be nice to see a comment here for what is going on here, and why it is needed.